checkrebuild

```diff
@@ -4,51 +4,168 @@
 
 shopt -s nullglob
 shopt -s extglob
+[ -t 1 ] && declare _terminal_output=1
 
 log="$(mktemp -d)"
+touch "$log/resolved_symbols"
 trap 'rm -rf "$log"' EXIT
 
 verbose=0
-while getopts "v" OPTION; do
+while getopts "vs" OPTION; do
     case $OPTION in
-        v) verbose=1 ;;
+        s) system=1 ;;
```

Comment:
Instead of a special "system" flag, let's make an argument that accepts a list of repos to skip (you can keep the `-s` as flag), and default it to `testing core extra community-testing community multilib-testing multilib`. Then if you have other repos that you don't want to check (such as blackarch or manjaro or whatnot), you pass your own set of repos. If you want to make a full system check, you pass an empty string.

Or maybe we could just introduce an environment variable to define these repos, if parsing them out from CLI args is not trivial.

I was considering a config file too, but I think it's not really needed if all we can configure is this list of repos :)

Reply:
Let me make a counter-proposal.
* no flag for testing only `aurutils+foreign` packages
* `-i repo_name` for testing `aurutils+foreign+repo_name` packages
* `-a` flag for testing whole system
Hard codding repository names seams short sighted. Arch is going to split their repos into smaller ones, and more and more Arch base distros is coming with they own repo naming scheme.